fastlane/Fastfile

```diff
@@ -26,14 +26,14 @@ platform :ios do
       
     run_tests(
       workspace: "OneLogin.xcworkspace",
-      devices: ["iPhone 14", "iPhone SE (3rd generation)"],
+      device: "iPhone SE (3rd generation)",
       result_bundle: true,
       scheme: options[:scheme],
       testplan: options[:testplan]
     )
     
     sh(
-      "bash #{options[:workspace]}/scripts/xccov-to-sonarqube-generic.sh #{options[:workspace]}/fastlane/test_output/OneLogin.xcresult > #{options[:workspace]}/fastlane/test_output/sonarqube-generic-coverage.xml"
+      "bash #{options[:workspace]}/scripts/xccov-to-sonarqube-generic.sh #{options[:workspace]}/fastlane/test_output/OneLoginStaging.xcresult > #{options[:workspace]}/fastlane/test_output/sonarqube-generic-coverage.xml"
```

Comment:
Should we make `OneLoginStaging` a variable here? ie. `#{options[:workspace]}/fastlane/test_output/[:scheme].xcresult`

Reply:
very good shout, not sure what determines what our `.xcresult` file name but I suspect it is the scheme, have changed this in my last PR and well try to confirm that it does correspond directly to the scheme name 🙂